configuration/scripts/machines/env.cheyenne_gnu

```diff
@@ -0,0 +1,23 @@
+#!/bin/csh -f
+
+source /glade/u/apps/ch/opt/lmod/7.2.1/lmod/7.2.1/init/csh
```

Comment:
Just curious, you seem not to use the 
```csh
set inp = "undefined"
if ($#argv == 1) then
  set inp = $1
endif

if ("$inp" != "-nomodules") then
```

bit it the icepack env files, in contrast to what's done in CICE.. 

Not that much of a deal but thought I'd mention.

Reply:
It's a reasonable question.  I have not migrated all the features from CICE scripts to Icepack.  Another one missing is the reuse of the binary during testing.  I think we can add those feature to Icepack scripts as it seems important.  We do a lot more testing and run a lot more tests in CICE than in Icepack at this time, so have prioritized the CICE scripts with performance upgrades.  I try to keep features (like ICE_CPPDEFS) consistent between the two as best as I can.